framework/utils/TreeStructure.py

```diff
@@ -321,30 +321,34 @@ def addChild(gNode,tNode,depth=0):
 ###########
 # PARSERS #
 ###########
-class XMLCommentParser(ET.XMLTreeBuilder):
-  """
-    An XML parser that expands on the default to preserves comments
-  """
-  def __init__(self):
-    """
-      Constructor.
-      @ In, None
-      @ Out, None
-    """
-    ET.XMLTreeBuilder.__init__(self)
-    self._parser.CommentHandler = self.handleComment
-
-  def handleComment(self,data):
-    """
-      Constructor.
-      @ In, data, string object to parse into comment
-      @ Out, None
-    """
-    self._target.start(ET.Comment,{})
-    self._target.data(data)
-    self._target.end(ET.Comment)
-
-#set up a parser for this module
+if sys.version_info.major > 2:
+  #In Python3, XMLParser is a C implementation, and can't be modified.
+  XMLCommentParser = ET.XMLParser
```

Comment:
Yes, we discussed that in https://hpcgitlab.inl.gov/idaholab/raven/issues/827 but that doesn't seem to have moved to github.